Documentation/configuration/verify-image-signatures.rst

```diff
@@ -0,0 +1,46 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _verify_image_signatures:
+
+**************************
+Verifying Image Signatures
+**************************
+
+Prerequisites
+=============
+
+You will need to `install cosign`_.
+
+.. _`install cosign`: https://docs.sigstore.dev/cosign/installation/
+
+Verify Signed Container Images
+==============================
+
+Since version 1.13, the Cilium container images are signed using cosign.
+
+Let's verify a Cilium image's signature using the ``cosign verify`` command:
+
+.. code-block:: shell-session
+
+    $ COSIGN_EXPERIMENTAL=1 cosign verify --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --certificate-github-workflow-name "Image Release Build" --certificate-github-workflow-ref refs/pull/[PR NUMBER]/merge quay.io/cilium/cilium:v1.13 | jq
```

Comment:
`--certificate-github-workflow-ref refs/pull/[PR NUMBER]/merge` should be `--certificate-github-workflow-ref refs/tags/[RELEASE TAG]`